src/libraries/System.Private.CoreLib/src/System/Collections/Generic/Dictionary.cs

```diff
@@ -122,12 +122,12 @@ private void Copy(IEnumerable<KeyValuePair<TKey, TValue>> collection)
                     return;
                 }
 
-                Debug.Assert(source._buckets is not null);
+                // This is not currently a true .AddRange as it needs to be an initalized dictionary
```

Comment:
```suggestion
                // This is not currently a true .AddRange as it needs to be an initialized dictionary
```

Reply:
I can never spell that word; in particular, don't know why